### PR TITLE
Remove unused targets

### PR DESCRIPTION
**Pull request by QuentinBisson**
## Checklist

I have:

- [x] Described why this change is being introduced
- [x] Separated out refactoring/reformatting in a dedicated PR
- [x] Updated changelog in `CHANGELOG.md`

### CODE REVIEW

File: files/templates/scrapeconfigs/additional-scrape-configs.template.yaml [hunk 1]
```diff
@@ -163,35 +163,6 @@
     target_label: app
 [[ include "_common" . | indent 2 ]]
 [[ include "_labelingschema" . | indent 2 ]]
-# aws-node
```
Comment (QuentinBisson) [inline files/templates/scrapeconfigs/additional-scrape-configs.template.yaml:166]: Moved to a servicemonitor

File: files/templates/scrapeconfigs/additional-scrape-configs.template.yaml [hunk 2]
```diff
@@ -674,7 +645,7 @@
   - action: labeldrop
     regex: label_giantswarm_io_machine_pool|label_giantswarm_io_machine_deployment
 [[ if eq .ClusterType "management_cluster" ]]
-# cert-operator (missing label)
+# cert-operator (missing label) (versionned so it needs to be kept)
```
Comment (QuentinBisson) [inline files/templates/scrapeconfigs/additional-scrape-configs.template.yaml:648]: That's dependent  on the release

File: files/templates/scrapeconfigs/additional-scrape-configs.template.yaml [hunk 3]
```diff
@@ -715,83 +686,6 @@
     action: drop
 [[ include "_common" . | indent 2 ]]
 [[ include "_labelingschema" . | indent 2 ]]
-# fluentbit
-- job_name: [[ .ClusterID ]]-prometheus/fluentbit-[[ .ClusterID ]]/0
```
Comment (QuentinBisson) [inline files/templates/scrapeconfigs/additional-scrape-configs.template.yaml:719]: That's not used

File: files/templates/scrapeconfigs/additional-scrape-configs.template.yaml [hunk 4]
```diff
@@ -715,83 +686,6 @@
...
-    replacement: $1
-    action: replace
-  - source_labels: [__meta_kubernetes_pod_label_app]
-    target_label: app
-  - source_labels: [__meta_kubernetes_service_annotationpresent_giantswarm_io_monitoring, __meta_kubernetes_service_labelpresent_giantswarm_io_monitoring]
-    regex: .*(true).*
-    action: drop
-[[ include "_common" . | indent 2 ]]
-[[ include "_labelingschema" . | indent 2 ]]
-# giantswarm-api (missing label)
```
Comment (QuentinBisson) [inline files/templates/scrapeconfigs/additional-scrape-configs.template.yaml:757]: This does not exist

File: files/templates/scrapeconfigs/additional-scrape-configs.template.yaml [hunk 5]
```diff
@@ -933,8 +827,8 @@
 [[ include "_apiserver" . ]]
 [[ include "_tlsconfig" . | indent 2 ]]
   relabel_configs:
-  - source_labels: [__meta_kubernetes_service_annotationpresent_giantswarm_io_monitoring]
```
Comment (QuentinBisson) [inline files/templates/scrapeconfigs/additional-scrape-configs.template.yaml:936]: Fix duplicate scrapes

**Review by hervenicol — APPROVED** [review]
I trust you for having the right servicemonitors to replace these scrapeconfigs :)

**Pull request merged**